src/chromeDebugAdapter.ts

```diff
@@ -353,26 +360,17 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         this.events.emitStepStarted('LaunchTarget.LaunchExe');
         const platform = coreUtils.getPlatform();
         if (platform === coreUtils.Platform.Windows && shouldLaunchUnelevated) {
-            const semaphoreFile = path.join(os.tmpdir(), 'launchedUnelevatedChromeProcess.id');
-            if (fs.existsSync(semaphoreFile)) { // remove the previous semaphoreFile if it exists.
-                fs.unlinkSync(semaphoreFile);
-            }
-            const chromeProc = fork(getChromeSpawnHelperPath(),
-                [`${process.env.windir}\\System32\\cscript.exe`, path.join(__dirname, 'launchUnelevated.js'),
-                semaphoreFile, chromePath, ...chromeArgs], {});
-
-            chromeProc.unref();
-            await new Promise<void>((resolve, reject) => {
-                chromeProc.on('message', resolve);
-            });
-
-            const pidStr = await findNewlyLaunchedChromeProcess(semaphoreFile);
+            let chromePid: number;
 
-            if (pidStr) {
-                logger.log(`Parsed output file and got Chrome PID ${pidStr}`);
-                this._chromePID = parseInt(pidStr, 10);
+            if (this._doesHostSupportLaunchUnelevatedProcessRequest) {
+                chromePid = await this.spawnChromeUnelevatedWithPineZorro(chromePath, chromeArgs);
```

Comment:
Would this be equivalent to use this._chromePID here instead?

Reply:
Fixed.